custom_dict.txt

```diff
@@ -126,8 +127,8 @@ formfeed
 fromlineno
 fullname
 func
-functiøn
 functiondef
+functiøn
```

Comment:
What? haha

Reply:
Yeah should probably remove that instead of sorting it alphabetically 😅 .

Reply:
It's used extensively in the unittest for non ascii name